lms/djangoapps/shoppingcart/tests/test_views.py

```diff
@@ -1488,9 +1488,34 @@ def test_shopping_cart_navigation_link_in_microsite_courseware_page(self):
             self.assertEqual(resp.status_code, 200)
             self.assertNotIn('<a class="shopping-cart"', resp.content)
 
+@attr(shard=3)
+@patch.dict('django.conf.settings.FEATURES', {'ENABLE_PAID_COURSE_REGISTRATION': True})
+@ddt.ddt
+class ShoppingCartViewDeletedCourseTests(SharedModuleStoreTestCase, XssTestMixin):
+    """
+    Test shopping cart view deleted course under various states
```

Comment:
The description is not clear enough could you please rephrase it to make more meaningful.